helix/principles/file-structure/modules.rst

```diff
@@ -7,13 +7,18 @@ Each module has the following structure:
 
     /…
         /[Module Name]     // Module root folder, named after the module (without prefixes)
-            /code          // Houses the main code for the module, for example the Visual Studio project with the module business logic or views.
+            /website       // Houses the website code for the module, for example the Visual Studio project with the website business logic or views.
             /serialization // Contains the serialized data from Sitecore, for example templates, content or security data
             /tests         // Unit or other test types for the module
+            /engine        // Houses the code for the plugin that runs on the Commerce Engine
```

Comment:
Please call this folder /commerce rather than /engine. 

Also add folders:
/xconnect              // Houses the code for plugins targeting the xConnect website
/client                    // Houses jss client-side code

Reply:
Agree with change to 'commerce' and inclusion of other folders, makes sense.

"abstractions" could be needed by probably needs further discussion outside of the scope of this PR

Reply:
I think we should remove `jss` as well for now? JSS and Helix is a bit more complicated of a subject, and including client side code per-module, intermixed with the server-side code, makes sense for some teams but may not work with the workflow of most teams. Also requires a more complicated build setup for the client-side code that we aren't prepared to demonstrate at this time.